Updated Okta Verify Push Verify api for number matching challenge (#1065)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/authn/index.md

}
```

##### Response example (waiting for 3-number verification challenge response)

> > **Note:** If Okta detects an unusual sign-in attempt then end user will be prompted for 3-number verification challenge and the correct answer of the challenge will be provided in the polling response. This is similar to standard `waiting` response but with an addition of `correctAnswer` field in the `challenge` object. `correctAnswer` will only be included in the response when end user is at 3-number verification challenge view on Okta Verify mobile app. Look at [Sign in to your org with Okta Verify](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-ov-signin) for more details about 3-number verification challenge flow.

Reviewer: ```suggestion
> **Note:** If Okta detects an unusual sign-in attempt, the end user will receive a 3-number verification challenge and their reply will be provided in the polling response. This is similar to the standard `waiting` response but with an addition of a `correctAnswer` property in the `challenge` object. The `correctAnswer` property will only be included in the response if the end user is on the 3-number verification challenge view in the Okta Verify mobile app. Look at [Sign in to your org with Okta Verify](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-ov-signin) for more details about this challenge flow.
```

Author: Thank you @jakubvul for the feedback! I have addressed your feedback with one exception, it's not the end user's reply that will be included in the response but the correct answer of the challenge.
